PupDates-CrossPol/pupdates_frontend
# Nav setup

#### What's this PR do?

Sets up navigation from the login screen (initial default view) to a home component, which for now is just an image of the logo.

#### Where should the reviewer start?

in App.js

#### How should this be manually tested?

Run ```npm install``` as a few dependencies were added, then ```npm start``` to make sure everything looks as expected. The styling for the login form is a bit off while we figure out how to hide the navigation bar that we think comes from the navigation packages.

#### Any background context you want to provide?

Order matters for the objects that define the various screens, so they should not be altered at this time.
